src/components/Kv/KvVerificationCodeInput.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<div
+		class="kv-verification-code-input"
+		:style="cssVars"
+	>
+		<!--
+			How I imagine this to work with multiple inputs:
+			multiple <inputs hidden from screenreaders
+			1 input hidden from sighted with the autocomplete tag.
+			the values get updated in sync.
```

Comment:
That makes sense! I think the tricky part will be focusing on the correct input, specifically when deleting/backspacing.

Reply:
oh good call, I hadn't thought about backspacing